api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/DQACargaViralCohortQueries.java

```diff
@@ -85,4 +116,145 @@ public DataDefinition getAge(String calculateAgeOn) {
     spdd.setQuery(substitutor.replace(sql));
     return spdd;
   }
+
+  public DataDefinition getDataNotificouCV() {
+
+    SqlPatientDataDefinition spdd = new SqlPatientDataDefinition();
+
+    spdd.setName("Data de Consulta onde Notificou o Resultado de CV");
+    spdd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+    spdd.addParameter(new Parameter("endDate", "End Date", Date.class));
+    spdd.addParameter(new Parameter("location", "Location", Location.class));
+
+    Map<String, Integer> map = new HashMap<>();
+
+    map.put("856", hivMetadata.getHivViralLoadConcept().getConceptId());
+    map.put("1305", hivMetadata.getHivViralLoadQualitative().getConceptId());
```

Comment:
You can remove this concept, It's never used

Reply:
All the concepts are being used in the code below